ccdproc/core.py

```diff
@@ -1740,6 +1741,94 @@ def _sigma_mask(baseline, one_sigma_value, lower_sigma, upper_sigma):
     return mask
 
 
+def bitmask2mask(bitmask, ignore_bits=0, flip_bits=None):
```

Comment:
BTW, masks and bitfields look like very similar things except for the role that each one plays in "applying mask": bits in a [mask](https://en.wikipedia.org/wiki/Mask_(computing)) indicate what bits in the bit field (=DQ field/value) should be modified/masked - at least that's my understanding.